rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -16,44 +16,262 @@
     private final static int TIMEOUT_IN_MS = 10000;
     private long rtmpPointer = 0;
 
+    private int rtmpError = RtmpIOException.RTMP_ERROR_NONE;
```

Comment:
This doesn't look like it's used for anything. Remove if so.

Reply:
Done.